pyproject.toml

```diff
@@ -16,7 +16,8 @@ license = "Apache-2.0"
 license-file = "LICENSE"
 dependencies = [
     "importlib-metadata; python_version<'3.8'",
-    "appdirs",
+    "matplotlib",
```

Comment:
Please remove this dependency in favor of vendoring by merging my branch

Reply:
Thanks for this! I've merged your branch, but now I am not sure how I supposed to import the `font_manager` module in the code.

Reply:
I've managed with that. I also wasn't fond of having to import the whole bloat of `matplotlib`, however, I wasn't aware of a better solution. This vendoring mechanism sounds nice!